compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 The Android Open Source Project
+ * Copyright 2022 The Android Open Source Project
```

Comment:
To maintain binary backward compatibility, we should keep names of classes the same. Adding this annotation probably will help, not sure (we need to check the generated bytecode):
```
@file:JvmName("DesktopPopup.desktop")
```

Reply:
We have to do that, because compiled libraries point to Java classes, not to the original Kotlin sources, where everything is the same.